data/net/minecraft/server/ChainedJsonException.mapping

```diff
@@ -1,4 +1,9 @@
 CLASS net/minecraft/server/ChainedJsonException
+	METHOD <init> (Ljava/lang/String;)V
+		ARG 1 message
+	METHOD <init> (Ljava/lang/String;Ljava/lang/Throwable;)V
+		ARG 1 message
+		ARG 2 parent
```

Comment:
This should be `cause`, following the conventional naming seen in exception constructors.
```suggestion
		ARG 2 cause
```